lib/ssb-cap.js

```diff
@@ -2,7 +2,7 @@
 //this will be updated whenever breaking changes are made.
 //(see secret-handshake paper for a full explaination)
 module.exports =
-  new Buffer('1KHLiKZvAvjbY1ziZEHMXawbCEIM6qwjCDm3VYRan/s=', 'base64')
+  new Buffer('P6EGPtCNW7irtdeIk+vRVzVbWOlctUKJuce1IZkO2N4=', 'base64')
```

Comment:
Resistence is futile! This looks like it was not supposed to be in the PR @theproductiveprogrammer :-)

Reply:
WhoopS! Thanks @arj03 - Fixed!